### PR TITLE
configuration file and CLI arguments doc

### PR DESCRIPTION
**Pull request by Geal**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -23,16 +23,15 @@ We'd encourage you to experiment with it, report troubles and offer your feedbac
...
 specified via flag, either by an absolute path, or a path relative to the current
 directory.
 
 ```
 OPTIONS:
-    -c, --config <configuration-path>    Configuration file location [env:
-                                         CONFIGURATION_PATH=]
-    -s, --supergraph <supergraph-path>   Supergraph Schema location [env: SUPERGRAPH_PATH=]
+    -c, --config <configuration-path>    Configuration file location
+    -s, --supergraph <supergraph-path>   Supergraph Schema location
```
Comment (BrynCooke) [inline README.md:34]: This section is missing the `--schema` option.

Comment (Geal) [reply README.md:34]: done in [4950770](https://github.com/apollographql/router/pull/668/commits/495077028c15a6895227fd0a492105ced7e47e0e)

File: docs/source/configuration/overview.mdx [hunk 2]
```diff
@@ -110,177 +141,76 @@ One you have generated the schema configure your text editor. Here are some inst
...
 
-The router sends a `TraceParent` header to subgraphs so they can create and send their own spans under the same trace.
+The Apollo Router supports [OpenTelemetry](https://opentelemetry.io/), with exporters for
+[Jaeger](https://www.jaegertracing.io/) and the
+[OpenTelemetry Protocol (OTLP)](https://aws-otel.github.io/docs/components/otlp-exporter)
+over HTTP or gRPC. The router generates traces that include the various phases of executing
+a GraphQL operation and associated dependencies.
 
-If the router receives an incoming request that already contains a `TraceParent` header, it uses that header's trace ID instead of generating a random one.
+You can set it up in the configuration file's [telemetry section](../telemetry).
```
Comment (BrynCooke) [inline docs/source/configuration/overview.mdx:180]: The telemetry page has gone, point to `open-telemetry`

Comment (Geal) [reply docs/source/configuration/overview.mdx]: done in f289eb3

**Review by BrynCooke — APPROVED** [review]

File: docs/source/configuration/cors.mdx [hunk 3]
```diff
@@ -6,18 +6,31 @@ description: Adjusting the cross-origin resource sharing (CORS) settings
...
+to indicate from which origin it accepts requests. **By default, it accepts requests from all origins**.
+If your environment has specific security requirements around CORS, they can be set up in the configuration file:
 
-You can configure CORS in the [YAML configuration file](#configuration-file) you provide to the Apollo Router.
-
-Here's an example configuration for adding `localhost` to the list of approved origins:
-
-```yaml
-# examples/nodejs/configuration.yaml
+```yaml title="configuration.router.yaml"
```
Comment (abernix) [inline docs/source/configuration/cors.mdx:13]: `config.router.yaml`.

**Review by abernix — APPROVED** [review]
Just the one comment below, but otherwise LGTM.

File: docs/source/configuration/cors.mdx (hunk 3 above)
Comment (Geal) [reply docs/source/configuration/cors.mdx]: done in [dc924f0](https://github.com/apollographql/router/pull/668/commits/dc924f050782fb2cab6e9ab1c6473ef7bed2b65c)

**Pull request merged**